### ISSUE TITLE
Issues with using inside of self-sizing tableview cell

### ISSUE THREAD

**Issue by StainlessStlRat** [vlas-voloshin/SubviewAttachingTextView#10]
I'm having a really weird issue. I'm using auto layout with a self sizing cell. The first time the cell displays, it is always the default height (44). But if I do anything to cause that cell to reload, i.e., cause some event that causes
     tableView performBatchUpdates... to be called or scroll around until the cell gets reloaded, it correctly resizes itself and everything is beautiful.  

I've tried so many things that I finally gave up and went with dynamically figuring out the height of the cell and returning that via tableView cell's delegate heightForRowAtIndexPath function. BUT, I STILL get the issue!

Basically, to circumvent it, I created a "minHeight" function. The minHeight function is very simple:

`@objc public func minHeight() -> CGFloat {
        self.textView.sizeToFit()
        self.textView.layoutIfNeeded()
        return self.textView.frame.height
    }`

This makes produces another set of results but ends in the same weird behavior.

In this specific case it should be three lines of "text" high, but on the first call to minHeight, it reports something approximately 2 lines in height. BUT yet again, when the cell is reloaded it correctly gives the necessary height.

Any ideas???

**Comment by StainlessStlRat** [comment]
Bug in autolayout/purelayout
